flake-module.nix

```diff
@@ -60,7 +60,9 @@ devenvFlake: { flake-parts-lib, lib, inputs, ... }: {
                 { "${shellPrefix shellName}container-${containerName}" = container.derivation; }
```

Comment:
Didn't know we had this in the flakes module. This should be exposed in the flake shim.